dockers/pipecd-base/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:alpine3.13
+
```

Comment:
Let's use `ARG` to define the using version of google sdk. (`ARG GOOGLE_CLOUD_SDK_VERSION=324.0.0` )